meta/devices/friendlyarm_nanopi-r1.yaml

```diff
@@ -0,0 +1,43 @@
+vendor: FriendlyARM
+model: NanoPi R1
+bootloader: unknown
+cpu:
+  model: Allwinner H3
+  cores: 4
+  mhz: 1200
+memory_mb:
+  - 512
+  - 1024
+storage:
+  - type: microSD
+device_type: WiFi Router
+usb:
+  - type: A
+    ports: 2
+ethernet:
+  - speed_mbit: 100
+    ports: 1
+    label: lan
+power_supply:
+  type: DC
+  volt: 5
+  ampere: 2
+leds:
+  - color: red
+    label: status
```

Comment:
The precise term is "function" instead of "label", at least this is what they use for the new led concept upstream.

Reply:
```suggestion
    function: status
```

Reply:
Yes.